### ISSUE TITLE
[BUG] Steam warning about starting FS25 with custom "" parameters, where there are no custom parameters set.

### ISSUE THREAD

**Issue by Theovorn** [FSGModding/FSG_Mod_Assistant#278]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

When I launch FS25 from the Mod Assistant, Steam warns me about trying to launch FS25 with "" custom parameters.
I have not set custom parameters in the Mod Assistant.

### Environment

```markdown
- Mod Assistant Version: 5.1.5
- OS: Win 10
```

### Anything else?

It seems to me that _for some reason_ the field is not considered empty even if it has no value set.

**Comment by jtsage** [comment]
this is a tough one, because try as I might, I can't duplicate it.  When 6.x hits, try hitting the new "clear settings" button on the version this is happening on and see if that helps.  Steam is funky sometimes. 

My more honest recommendation is to just enable -cheats in both places - there are some very useful options that enables in the console, and there is no downside - the Giants devs were nice enough to not disable achievements when active even.

I'm gonna add this to my master "still might need fixed" list and close this issue, I'm kinda drowning in todo's at the moment.

**Comment by vagtj** [comment]
I just got FS25, and I got the same issue even with trying to enable cheats in both places. Just out of curiosity on the you can not replicate this error, is your game installed on the c: drive or is it on another drive? My game is not on main drive.

**Comment by Theovorn** [comment]
My game is not installed on the C: drive.
If I add the custom parameters in both places, it makes no difference, I still get the annoying Steam warning.

**Comment by vagtj** [comment]
I am wondering if that has something to do with why we keep getting that annoying argument. FS22 and 19 work for me perfectly. I also don't get why mine tries to open steam every time. By chance where did you buy your game/activation code? Was it through steam, giants, or third party?

**Comment by jtsage** [comment]
Yeah, this is part of my issue with reproducing.  I have FS19 on steam, with no issues - but 22 and 25 I bought from the eshop, so they never touch steam.  I'll give installing FS19 on a different drive and see if that makes a difference in being able to make this happen.

**Comment by jtsage** [comment]
So, some more information from my FS19 install - I added all the command line options I wanted to steam, and then cleared the list in MA - it works, the options are applied, and I get no popup.  I'd love somebody to see what happens with 22 and/or 25 using this method - it's a stupid workaround, but everything I've found online tells me this is a common, and pretty much unsolvable problem with any sort of mod manager (many, many games listed)

**Comment by vagtj** [comment]
I just tried adding arguments in steam and launched the game. Went back into mod manager and the arguments I put in steam did not show up there to clear out

**Comment by vagtj** [comment]
So, when I add your custom buttons for the game arguments, the " " show up as well. Please see added screen shots. 
![Screenshot 2025-01-15 200016](https://github.com/user-attachments/assets/056d44e8-ef01-44e0-8daa-cb0b6fa90f87)
![Screenshot 2025-01-15 195815](https://github.com/user-attachments/assets/3f5f231e-9ab9-4d00-b6c0-c90a1c3cfd60)
But when I hand type them in, no " ". See attached additional files.
![Screenshot 2025-01-15 200947](https://github.com/user-attachments/assets/52fa90b3-7685-4919-9760-99d050926f0c)
![Screenshot 2025-01-15 201005](https://github.com/user-attachments/assets/29bb9e4a-05de-4918-a0ed-bc52feef6b81)
Maybe this can help you.

Thanks,
Jason

**Comment by vagtj** [comment]
It is as if there maybe a double " attached to the "-cheats buttons"

**Comment by Dylan-M** [comment]
> I just tried adding arguments in steam and launched the game. Went back into mod manager and the arguments I put in steam did not show up there to clear out

It won't show up in FSG MA, but should still **work** if they are set in Steam and the game is launched from MA.

**Comment by mndooley** [comment]
If you can think of a way that I can stream to you I would be more than willing to show you and basicly use me as a test bed. I can recreate it like clockwork and I have it on my C drive. Its getting real annoing.

**Comment by nocosimfarmer** [comment]
I am have removed game arguments from both MA and Steam and I still get the "" error when trying to launch with MA. No issues when launching from Steam.
